### PR TITLE
trouble-shooting: update the issue report link

### PR DESCRIPTION
**Pull request by CaitinChen**
Via: https://github.com/pingcap/docs/pull/1091
@lilin90 PTAL

### CODE REVIEW

File: trouble-shooting.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ category: advanced
 
 # TiDB 集群故障诊断
 
-当试用 TiDB 遇到问题时，请先参考本篇文档。如果问题未解决，请按文档要求收集必要的信息通过 Github [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new)。
+当试用 TiDB 遇到问题时，请先参考本篇文档。如果问题未解决，请按文档要求收集必要的信息通过 Github [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new?template=bug-report.md)。
```
Comment (lilin90) [inline trouble-shooting.md]:
```suggestion
当试用 TiDB 遇到问题时，请先参考本篇文档。如果问题未解决，请按文档要求收集必要的信息通过 Github [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new/choose)。
```

File: trouble-shooting.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ category: advanced
...
 
 ## 如何给 TiDB 开发者报告错误
 
-当使用 TiDB 遇到问题并且通过后面所列信息无法解决时，请收集以下信息并[创建新 Issue](https://github.com/pingcap/tidb/issues/new):
+当使用 TiDB 遇到问题并且通过后面所列信息无法解决时，请收集以下信息并[创建新 Issue](https://github.com/pingcap/tidb/issues/new?template=bug-report.md):
```
Comment (lilin90) [inline trouble-shooting.md]:
```suggestion
当使用 TiDB 遇到问题并且通过后面所列信息无法解决时，请收集以下信息并[创建新 Issue](https://github.com/pingcap/tidb/issues/new/choose):
```

File: trouble-shooting.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ category: advanced
 
 +   panic
 
-    程序有错误，请将具体的 panic log [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new)。
+    程序有错误，请将具体的 panic log [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new?template=bug-report.md)。
```
Comment (lilin90) [inline trouble-shooting.md]:
```suggestion
    程序有错误，请将具体的 panic log [提供给 TiDB 开发者](https://github.com/pingcap/tidb/issues/new/choose)。
```

**Comment by CaitinChen** [comment]
Thanks. PTAL again.

**Pull request merged**